.github/workflows/container-validation-backends.yml

```
@@ -0,0 +1,70 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+name: NVIDIA Github Validation
+
+on:
+  push:
+    branches:
+      - main
```

Should we add this for releases as well?
```suggestion
      - main
      - release/*
```

---

I would wanna avoid it in start, we can consider it once we have eks setup done. We always do PR for release branch, so should cover some of it